JavaDoc generation fails when plugin doesn't handle Vaadin dependencies

In our project Vaadin dependencies are handled manually so we have 

```
vaadin {
    manageDependencies = false
    manageRepositories = false
}
```

After updating to the latest 0.7.1 plugin, JavaDoc generation (`gradle javadoc`) fails with the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Configuration with name 'vaadin-javadoc' not found.
```

I workarounded the issue by adding the following to our build.gradle:

```
project.configurations.create("vaadin")
project.configurations.create("vaadin-javadoc")
```

--- a/src/main/groovy/fi/jasoft/plugin/TaskListener.groovy
+++ b/src/main/groovy/fi/jasoft/plugin/TaskListener.groovy
@@ -105,8 +105,12 @@ public class TaskListener implements TaskExecutionListener {
 
         if (task.getName() == 'javadoc'){
             task.source = Util.getMainSourceSet(project)
-            task.classpath += project.configurations[Configuration.JAVADOC.caption()];
-            task.classpath += project.configurations[Configuration.SERVER.caption()];
+            if (project.configurations.findByName(Configuration.JAVADOC.caption()) != null) {
+                task.classpath += project.configurations[Configuration.JAVADOC.caption()]
+            }
+            if (project.configurations.findByName(Configuration.SERVER.caption()) != null) {
+                task.classpath += project.configurations[Configuration.SERVER.caption()]
+            }
             task.failOnError = false
             task.options.addStringOption("sourcepath", "")
         }

